locales/en.json

```diff
@@ -1,16 +1,87 @@
+            },
+            "action": "Donate a cuppa"
+        },
+        "team": {
+            "header": {
+                "headline": "Meet the Team"
+            },
+            "moreContributors": {
+                "headline": "Plus many more members who have lent us a helping hand!",
+                "action": {
+                    "text": "Find them on our {frontend} and {localization} Github contributors pages",
+                    "frontend": "front end",
+                    "localization": "localization"
+                }
+            },
+            "join": {
+                "headline": "Does Find a Doc’s mission spark your interests?",
+                "subheadline": "Join us over on GitHub to see how you can contribute!",
+                "action": "I want to join!"
+            }
+        },
+        "timeline": {
+            "items": {
+                "0": {
+                    "date": "2021.06.17",
+                    "dateHeadline": "From idea to health database",
+                    "text": {
+                        "0": "On that day, LaShawn Toyoda had enough of the lack of direction from officials. A few hours later, Find a Doc is born.",
+                        "1": "This database aims to better circulate the latest information on healthcare services in Japan."
+                    },
+                    "textHeadline": "Watch our daughter, I gotta make something."
+                },
+                "1": {
+                    "date": "Presently",
+                    "dateHeadline": "A team of volunteers",
+                    "text": {
+                        "0": "Find a Doc is community-driven with the support of a team of volunteers. Currently, the team is focused on keeping the immigrant community informed about clinics offering waiting lists for vaccinations to reduce waste.",
+                        "1":  "In the future, we want Find a Doc to help non-Japanese speaking residents of Japan find clinics that offer services in their native language."
+                    },
+                    "textHeadline": "You can fill the void by citizen action.",
+                    "action": "Check out &/or join the team"
+                }
+            }
+        },
+        "trust": {
+            "cards": {
+                "0": {
+                    "text": {
+                        "0": "All links lead to official resources such as clinic and government websites.",
+                        "1": "Read them thoroughly before registering your personal information with them."
+                    }
+                },
+                "1": {
+                    "text": {
+                        "0": "Anyone in the community can submit content to the database.",
+                        "1": "As a result, is not possible for us to guarantee that accuracy. If you notice any errors, please report the issue to moderators."
```

Comment:
`As a result, is not possible for us to guarantee complete accuracy. If you notice any errors, please report them to moderators.`